src/LoggerFactory.ts

```diff
@@ -112,4 +120,9 @@ export class LoggerFactory {
     }
     return __slf._logLevel || level || (envLevel && Level[envLevel]) || Level.Debug;
   }
+
+  private static isEventBelowLevel(event: Event, level: Level = this.getLogLevel()): boolean {
+    const levelKey = capitalize(event.level);
+    return level > Level[levelKey];
+  }
```

Comment:
This doesn't need to be part of the class. It can be a function outside of the class.